tests/source/test_file.py

```diff
@@ -79,29 +88,43 @@ def test_args(self):
 
     def test_config_readonly_default(self):
         config = FileSource.config(
-            parse_unknown("--source-file-filename", "feedface")
+            parse_unknown(
+                "--source-file-filename",
+                "feedface",
+                "--source-file-label",
+                "default-label",
+            )
         )
         self.assertEqual(config.filename, "feedface")
+        self.assertEqual(config.label, "default-label")
         self.assertFalse(config.readonly)
 
     def test_config_readonly_set(self):
         config = FileSource.config(
             parse_unknown(
-                "--source-file-filename", "feedface", "--source-file-readonly"
+                "--source-file-filename",
+                "feedface",
+                "--source-file-label",
+                "default-label",
+                "--source-file-readonly",
             )
         )
         self.assertEqual(config.filename, "feedface")
         self.assertTrue(config.readonly)
```

Comment:
Check that label is being set to `default-label` here

Reply:
Done :+1: